.husky/post-checkout

```diff
@@ -0,0 +1 @@
+npx --yes -- git-pull-run@latest --install
```

Comment:
I think if `git-pull-run` is installed locally, the `npx --yes` is not necessary, right? I've been using `pnpm` yeself, so not not sure though.

Reply:
I think you are correct that if `git-pull-run` is installed locally then you could omit `--yes` (or use `--no` which should be the same result). Using `--yes` should use the latest published `git-pull-run` from npmjs. I don't think `@latest` is actually needed and there may have been a flaw or some confusion on my part during my testing.